user-interface/src/main/java/life/qbic/datamanager/views/general/icon/IconFactory.java

```diff
@@ -0,0 +1,24 @@
+package life.qbic.datamanager.views.general.icon;
+
+import com.vaadin.flow.component.icon.Icon;
+import com.vaadin.flow.component.icon.VaadinIcon;
+
+/**
+ * <b>Icon Factory</b>
+ *
+ * <p>Convenience icon factory for re-used icons following the iconography of the application's
+ * style guide</p>
+ *
+ * @since 1.7.0
+ */
+public class IconFactory {
+
+  private IconFactory() {}
+
+  public static Icon warningIcon() {
```

Comment:
not used yet. We should avoid introducing code that is not used. Hard, because one thinks about use cases during coding, but we must not. It just produces dead code.